frontend/src/AppRoutes/Private.tsx

```diff
@@ -1,13 +1,12 @@
 /* eslint-disable react-hooks/exhaustive-deps */
-import { notification } from 'antd';
 import getLocalStorageApi from 'api/browser/localstorage/get';
 import loginApi from 'api/user/login';
 import { Logout } from 'api/utils';
 import Spinner from 'components/Spinner';
 import { LOCALSTORAGE } from 'constants/localStorage';
 import ROUTES from 'constants/routes';
 import history from 'lib/history';
-import React, { useEffect, useMemo } from 'react';
+import React, { useContext, useEffect, useMemo } from 'react';
```

Comment:
rather than importing useContext everywhere can we make a custom hook ?

Reply:
Hi @palashgdev, thank you for the review. Making custom hooks makes sense, will work on the changes and update the PR